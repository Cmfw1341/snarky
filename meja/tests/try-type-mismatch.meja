let x = fun (b) => {
  try ({
    if (b) {
      ();
    } else {
      failwith("");
    };
  }) {
    | Failure(_) => b
  };
};
