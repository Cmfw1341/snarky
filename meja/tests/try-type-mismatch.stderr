File "tests/try-type-mismatch.meja", line 9, characters 20-21:
Error: Incompatible types unit and bool:
       Cannot unify unit and bool
